content/building-blocks/prepare-your-data-for-analysis/data-preparation/large-datasets-R.md

```diff
@@ -2,21 +2,25 @@
 title: "Import Large Datasets Into R"
 description: "Importing large datasets in R is made easy by data.table R package. The package makes importing and manipulating huge datasets (like big data) in R faster"
 weight: 2
-keywords: "import, data, data preparation, big data, large datsets, memory, RAM, data.table, big data, R"
+keywords: "import, data, data preparation, big data, large datsets, memory, RAM, data.table, big data, R, dataframe, object size"
 date: 2021-01-06T22:01:14+05:30
 draft: false
 aliases:
   - /import/large-datsets
 ---
 
 ## Overview
-
-Many R-users rely on the `dplyr` or `read.table` packages to import their datasets as a dataframe. Although this works well for relatively small datasets, we recommend using the `data.table` R package instead because it is significantly faster. This building block provides you with some practical tips for dealing with large datsets in R.
+Many R-users rely on the `dplyr` or `read.table` packages to import their datasets as a dataframe. Although this works well for relatively small datasets, we recommend using the `data.table` R package instead because it is significantly faster. This building block provides you with some practical tips for dealing with large datasets in R.
 
 ## Code
```

Comment:
I would change subtitle to "Import Dataset Using `data.table` package" or something shorter and succinct instead of just "Code"

Reply:
Good Idea! I've substituted it to "Using `data.table` package to import a Dataset"